department-of-veterans-affairs/abd-vro
# Moving VRO App

## What was the problem?

The VRO App application was no longer providing any functionality being used.

Associated tickets or Slack threads:
- #2697 

## How does this fix it?[^1]

Moves the VRO app application to its own repo https://github.com/department-of-veterans-affairs/abd-vro-app

## How to test this PR
- Make sure nothing breaks in local builds 
- Remove the VRO app in escalating environments ensuring there are no disruptions to the function of other services 


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
